modules/candidate_parameters/php/module.class.inc

```diff
@@ -42,6 +42,10 @@ class Module extends \Module
                 return [];
             }
 
+            if (\NDB_Config::singleton()->getSetting('useConsent') != 'true') {
```

Comment:
The settingEnabled function seems like it would be a little more robust.

Reply:
okay, in which class would you want the consentEnabled() defined?

Reply:
php/libraries/Settings.class.inc seems like the most logical place